jujugui/views.py

```diff
@@ -74,6 +74,7 @@ def config(request):
     env_uuid = settings.get('jujugui.uuid', 'sandbox')
 
     charmstore_macaroons = settings.get('jujugui.charmstore_macaroons')
```

Comment:
Do we still need the charmstore_macaroon to be passed via settings, or with the idm token can we handle all the login stuff like we do in HJC?